cmd/juju/commands/import_sshkeys.go

```diff
@@ -74,6 +74,14 @@ func (c *importKeysCommand) Init(args []string) error {
 		return errors.New("no ssh key id specified")
 	default:
```

Comment:
:x: Can you drop the switch and just do

```go
if len(args) == 0 {
    return errors.new("no ssh key id specified")
}
// rest of code
```

The like 4 level deep indenting is ugly a hard to read.